vector/src/main/res/values/strings.xml

```diff
@@ -842,8 +848,8 @@
     <string name="encryption_import_room_keys_summary">Import the keys from a local file</string>
     <string name="encryption_import_import">Import</string>
     <string name="encryption_import_enter_passphrase">Enter passphrase</string>
-    <string name="encryption_never_send_to_unverified_devices_title">Encrypt to verified devices only</string>
-    <string name="encryption_never_send_to_unverified_devices_summary">Never send encrypted messages to unverified devices from this device</string>
+    <string name="encryption_never_send_to_unverified_devices_title">Only message verified devices</string>
+    <string name="encryption_never_send_to_unverified_devices_summary">If a room has encryption enabled only send messages when all devices have been verified.</string>
```

Comment:
I think the meaning is not the same than before. Sorry I haven't seen this on my first review. Can you improve (or revert) these two wordings please?

Reply:
I thought it was more clear after the change, is my wording incorrect? This setting doesn't apply to unencrypted rooms as far as I know. Plus, the old description uses the word never which makes it a negation.

This is a hard one, `Don't never send encrypted messages` is my first thought when the setting is off with the old description. The use of `only` also makes it odd at first glance, but I left that in because I couldn't improve it.

Reply:
Yes, we think the new wording is incorrect.

Reply:
Fixed this, the only changes since yesterday are adding a period at the end of that description and matching the capitalization of `Not tagged` with the other room tag options.

Reply:
(Note that R.string.room_settings_tag_pref_no_tag is never used and could be removed)